README.md

```diff
@@ -148,7 +148,43 @@ For non-Java applications or tools like `cqlsh`, you can run the Spanner Cassand
 
     This will create an executable jar file `spanner-cassandra-launcher.jar` inside the folder `google-cloud-spanner-cassandra/target`.
 
-* Run the jar using the command:
+* Run the jar using one of the following methods:
+
+    **1. Using a YAML Configuration File (Recommended for Production)**
+
+    For production and complex setups, it is recommended to use a YAML file to configure the adapter. This method supports multiple listeners and global settings.
+
+    Create a `config.yaml` file with the following structure:
+    ```yaml
+    globalClientConfigs:
+      spannerEndpoint: "spanner.googleapis.com:443"
```

Comment:
I would remove `spannerEndpoint` from sample. This is mostly relevant for our testing, I do not expect customers to change this.